web/src/beta/components/fields/PropertyFields/index.tsx

```diff
@@ -19,6 +20,7 @@ const PropertyFields: React.FC<Props> = ({ propertyId, item }) => {
       {item?.schemaFields.map(sf => {
         const isList = item && "items" in item;
         const value = !isList ? item.fields.find(f => f.id === sf.id)?.value : sf.defaultValue;
+        const { min, max } = sf;
```

Comment:
Is this okay or should there be defaults for `0` and `100` for the min-max values?

Reply:
I don't think we should set defaults because we don't know what the slider field will be pointing to. 
min/max values should be set for things like the slider. If they aren't set, we should add them to the manifest in the backend 👍🏻 
Let me know if you are getting null/undefined for any values related to the slider 🙏🏻

Reply:
Umm..I don't know where they are being used so I don't know how to check!

Reply:
I think we used min/max once with spacing so we can define the min and max padding for an item (like image block ) for example

Reply:
Sorry can you remove this and just use min/max like other sf values (ie sf.min, sf.max)